### ISSUE TITLE
Marker popup is too big on mobile devices and takes up almost all of the map space

### ISSUE THREAD

**Issue by perfectly-preserved-pie** [perfectly-preserved-pie/larentals#7]
The map looks like dog shit on mobile devices due to the popup not dynamically resizing itself for the smaller screen size of mobile devices. It makes it really hard to view all information in the popup or navigate around the map when the popup is open. 

[Example on iOS Safari](https://user-images.githubusercontent.com/28774550/204154932-d7d41930-6d47-49d1-90c7-5ca619b6c03a.jpeg)

**Comment by perfectly-preserved-pie** [comment]
Can I maybe use one of the following `dl.Popup` properties to change the popup size?

```
- autoPanPadding (dict; optional):
    Equivalent of setting both top left and bottom right autopan
    padding  to the same value.

- autoPanPaddingBottomRight (dict; optional):
    The margin between the popup and the bottom right corner of the
    map view after autopanning was performed.

- autoPanPaddingTopLeft (dict; optional):
    The margin between the popup and the top left corner of the map
    view after autopanning was performed.```

**Comment by perfectly-preserved-pie** [comment]
Ok, I gotta attack this in chunks.

If the user agent is mobile:

1. The MLS photo needs to be resized; right now it's at 300x400, which is too big for mobile devices and is what dictates how _wide_ the popup is. We can resize it on the fly with ImageKit transformations.
2. For _height_, I need to add another column `mobile_popup_html` with the `table` property set to a specific pixel size, like `<table height="200"`. This will make the table scrollable if there's an overflow.
3. When I append the markers to the list, I need to have `popup=row.mobile_popup_html` instead of `popup=row.popup_html`

The resulting popups should be smaller on mobile devices only.

**Comment by perfectly-preserved-pie** [comment]
Ok, thanks to https://community.plotly.com/t/anyone-using-dash-leaflet-on-a-mobile-device-how-do-you-make-the-popup-size-appropriate-for-the-screen-size/74129/2?u=the.oldest.house, I found a way to resize the Leaflet popup using CSS rather than Dash-Leaflet itself. It looks decent now:

![image](https://user-images.githubusercontent.com/28774550/230743603-471ea0f9-f626-48ac-9c21-ee0fe9ab4cb9.png)

[I made a CSS file](https://github.com/perfectly-preserved-pie/larentals/blob/dev/assets/dash_leaflet_popup_style.css) that styles the Leaflet popup appropriately.

I'd still like to limit the vertical height and add a scrollbar, but for now this is a huge breakthrough.
